database/src/main/java/info/metadude/android/eventfahrplan/database/sqliteopenhelper/SessionsDBOpenHelper.java

```diff
@@ -54,63 +53,25 @@
                     Columns.CHANGED_DURATION + " INTEGER," +
                     Columns.CHANGED_IS_CANCELED + " INTEGER)";
 
+    private static final String SESSIONS_INDEX_CREATE =
+            "CREATE UNIQUE INDEX " + Indices.SESSION_ID + " ON " + SessionsTable.NAME + "(" + Columns.SESSION_ID + ")";
```

Comment:
Why not use an `id INTEGER PRIMARY KEY` column? It feels wrong to tie your database model too strongly to the network model.